db_stress_tool/expected_state.cc

```diff
@@ -406,6 +414,12 @@ class ExpectedStateTraceRecordHandler : public TraceRecord::Handler,
       return Status::Corruption("unable to parse key", key.ToString());
     }
 
+    bool should_buffer_write = (buffered_writes_ != nullptr);
+    if (should_buffer_write) {
+      return WriteBatchInternal::Delete(buffered_writes_, column_family_id,
```

Comment:
Nit: can we use `buffered_writes_->Delete()`?
`WriteBatchInternal` is internal, thus using it should be restricted or avoided.

Reply:
This applies to other operations.

Reply:
A quick question - I am not aware of `WriteBatch:: API` accepting `uint32_t column_family_id` (which is what we have here.) instead of `ColumnFamilyHandle* column_family`. Can we use internal API in this case?

Reply:
oh ok, then nvm.
It can be done, but using internal functions is also ok then.